cmd/git-security/api/repository_test.go

```diff
@@ -132,3 +139,240 @@ func TestArchiveRepoNormal(t *testing.T) {
 	require.Nil(t, err)
 	assert.Equal(t, 200, resp.StatusCode)
 }
+
+func TestSetSession(t *testing.T) {
```

Comment:
Why do we have this test? are you testing your logic or testing goFiber code? or the store? which is written by others.